Roychela/Politicalapp-APIs
# #163841768 Added procfile

**What does this PR do?**
It creates a procfile to enable app deployment on Heroku.

**Describe the tasks to be completed.**

- [x] Create Procfile.

- [x] Install gunicorn.

- [x] Add gunicorn to the requirements.txt 

**How can this be manually tested?**
Clone the repo. Navigate to the Politicalapp-APIs folder and check if the procfile is there and if gunicorn is in the requirements.txt file.

**What are the relevant Pivotal Tracker stories?**
https://www.pivotaltracker.com/story/show/163841768
